aireview/justfile

```diff
@@ -15,4 +13,45 @@ run:
     poetry run python src/aireview/main.py
 
 tests:
-    poetry run pytest
\ No newline at end of file
+    poetry run pytest
+
+prefect-login:
+    poetry run python prefect/deploy/login.py
+
+# See the help message by: just manage-block --help
+manage-block +args:
+    poetry run python prefect/deploy/manage_block.py {{ args }}
+
+# See the help message by: just manage-flow --help
+manage-flow +args: prefect-login
```

Comment:
**style**: The manage-flow and manage-flow-local recipes are duplicates with identical dependencies. Consider consolidating them into a single recipe with a flag parameter for local execution.